### DISCUSSION TITLE
Run in a Windows VM?

### DISCUSSION THREAD

**Discussion by MostHated** [wezterm/wezterm#2421] [General]
Hey all,
    I am running Windows in a VM, but attempting to open wezterm gives something along the lines of " Error, wezterm_gui frontend  Failed to create window". Is there some sort of mode/switch in which I could use to allow it to run in a VM? I recall in the past that I was able to get 3d application such as Houdini to work in a Win VM by setting some env vars before launch, but it has been quite some time and I don't recall what they were, or if those particular ones would even help in this situation. If anyone has any insight, I would appreciate it.

Thanks,
-MH

**Comment by MuhammedZakir** [comment]
Maybe related: https://github.com/wez/wezterm/issues/1813

**Comment by MostHated** [comment]
The info in there did at least get the terminal launched, so I do appreciate that. Unfortunately, doing it that way feels a bit sluggish, but I suppose that is just the nature of running things in a VM.
